src/main/kotlin/com/thelastpickle/tlpstress/Metrics.kt

```diff
@@ -36,6 +36,7 @@ class Metrics(metricRegistry: MetricRegistry, val reporters: List<ScheduledRepor
     val errors = metricRegistry.meter("errors")
     val mutations = metricRegistry.timer("mutations")
     val selects = metricRegistry.timer("selects")
+    val deletions = metricRegistry.timer("deletions")
```

Comment:
I hadn't considered the metrics, and at first I was against this, but I think you're right, explicitly tracking deletes as a separate metric is the right decision.